gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/PipelineController.groovy

```diff
@@ -60,10 +60,39 @@ class PipelineController {
   @Autowired
   ObjectMapper objectMapper
 
+  @CompileDynamic
   @ApiOperation(value = "Delete a pipeline definition")
   @DeleteMapping("/{application}/{pipelineName:.+}")
   void deletePipeline(@PathVariable String application, @PathVariable String pipelineName) {
-    pipelineService.deleteForApplication(application, pipelineName)
+    List<Map> pipelineConfigs = front50Service.getPipelineConfigsForApplication(application, true)
+    if (pipelineConfigs!=null && !pipelineConfigs.isEmpty()){
```

Comment:
Please run spotless:apply to fix the formatting

Reply:
This has been run already. Still ran this task, but no formatting changes seen.